src/wallet/rpcwallet.cpp

```diff
@@ -2298,6 +2298,14 @@ UniValue walletpassphrase(const JSONRPCRequest& request)
     // Alternately, find a way to make request.params[0] mlock()'d to begin with.
     strWalletPass = request.params[0].get_str().c_str();
 
+    // Get the timeout
+    int32_t nSleepTime = request.params[1].get_int();
```

Comment:
Probably should use `int`, not `int32_t` (they're not guaranteed to be the same)

Reply:
Done.